ai_transform/workflow/abstract_workflow.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         send_email: bool = True,
         success_threshold: float = 0.8,
         email: dict = None,
+        frontend_ctas: Union[DownloadCTA, LinkCTA, FrontendCTA, dict]= None,
```

Comment:
`FrontedCTAType = Union[DownloadCTA, LinkCTA, FrontendCTA, dict]`

Reply:
Why would you want this.